concent_api/verifier/tasks.py

```diff
@@ -186,13 +191,13 @@ def verification_result(
         subtask.full_clean()
         subtask.save()
 
-    elif result in (VerificationResult.MATCH, VerificationResult.ERROR):
+    elif result_enum in (VerificationResult.MATCH, VerificationResult.ERROR):
         # Worker logs the error code and message
-        if result == VerificationResult.ERROR:
+        if result_enum == VerificationResult.ERROR:
             logger.info(
                 f'verification_result_task processing error result with: '
-                f'SUBTASK_ID {subtask_id} -- RESULT {result} -- ERROR MESSAGE {error_message} -- ERROR CODE {error_code}'
-            )
+                f'SUBTASK_ID {subtask_id} -- RESULT {result_enum.name} -- ERROR MESSAGE {error_message} -- ERROR CODE {error_code}'
+            )  # pylint: disable=no-member
```

Comment:
Could you please disable this warning for this enum to `pylintrc`? I see that we're still using `pylint: disable` for enums and it's a really bad practice because it makes us used to disabling warnings rather than treating them as something that has to be fixed.